contracts/cosmos/light-client/scripts/data/bootstrap.txt

```diff
@@ -0,0 +1 @@
/0suRv1hdoIGKwcBVlEIBzNOryDK2USpRK9mTiWbXRrSy3emebCaa43f77wyki5A5F1Xx9V2eBmWkDdJTCdqfrMrCN1DBOlYDpVsBtVWZNrczENN3mTDHfum4sJ4Nni3IvZ0Dyl7PvQ8/Ra1iIrq45nCT+817LV2Drl7LQkbFSaVZqUdjLjvVkE6b9yQ9IErE5KhHLgyk2A1FqdxYTUJluNaBBf+KDFapYiLjKMy44il65zmyW+kWg2X+VDBLvB9gZtAf5gpqI2nkCj2EXxgQ2eB7GYw6UX/nI+dTXRystbz3Kxrryw3lJF/iOVGHQYbf3wCulST8zLoVXk0f0SZ1+DytDTHCBYXi7LUMQMd1AACnO/I5HE2B24LaeRuHEIJAWVd7icogt8GxOpCC8Diu+4aVQaGo2iekYbEV0v1s9zK4Q0c3IHdfMcysuLEVVYt2ex74YbqZmOdgXhIGsWZ0dfzFKkZ3i6dxKfrkdd/pYsZpTqBWDVhP+2L1JEkGGg2PkzR83jYTZwA5+3e/sUafeZnET1pelyN840H4fUi1Cz9TxnaPCqwClN1zpRBxuOBp0PNMW3Y+7zX6hOP6EaeNHeZlN5zlmFkNLJlitZJ2/5XpwJl7brZw8Q8EcViRU9skEehr6FehAWi9SZiASXNpOv0TuFESH2nhYkYeWfsQd2kGcBvSxoj3b4CYrFFQQNUAWrzgVZ+tnGRNhebwy9zCNbc7y1Zo9eXu6w4rPXGJvOdcZaZe7PRTX0mML+qxyTe62+nxwDoe2h18R/zTbf7VIFPNJrflDNRgoMeAuaDxfNUb9ftO0rjcaLJyTKCRTY+6nfIu1oCZtoEQNUHlBmSX+OFqCOy2C1QL7Sgz8JpiyNmq8at64FHs4b5gQwrIlV8avML8YEIKxSM4+ZiR8Qak8QMs4DFLcnNTIYpinhNdcVk28xsLCfegYy7i8dcE+3Dcb15I1zxyhXVDdBCQ8o7D52iJrcZATIkGrCrF9QUQMSr2+VtIpsm/iJt5n8TfI8cbn5TOGsFWo+BqDUhuJHed76agRpyU/fM/LncXOZhgMccpfV//Hyj0knzO7LODTbsvZO1+lVdc5XbEX943hVE3cA57ozBLrnN8YBi0CEIrUs12MgJUVW/hIarlgEsM9nMos1cClsghYg5MW8flYLTgaMJ/oUoyAAAOZXoMWRp41flk22pe1zCsm+ZwqKh6nhYKFusEcNcV6VlvhiFXOZRSzfNEHXhK0k326LWQXbjTvpsIV1654SpwBlGJzoEnABLmW89mYOPE6YKljhVWrs5/gc8Vt6vr5smM7FUB8NPrD4aphMykTorN47aNeRtoz4tb2NPfg//WHiMdqWOcQ/pvOcCGagBI+V4vLKm+Lc467Dzd8OKgN2wrxOltcmr8zzIgmYeLka3q0lmKWEryqr+AVDihaBlG2SqdKU0rhFq9ORuFdLuwxv+ejsDywoubhb8ZBDZTmSfQ8dZ7LTNz4+v9qAERbIeq5wV4rkwy1VRnDoDJ1Znm9N9KQutjI+3zoTnMUCGwov+S8yDHpsJSCohjC+lJZ8zH4ANCa7LflhWcH89XAcjbuyx5DyIVB3uf9WnuJc4MGFsFk0p+hd/OnN7edGjqVxnCcRt0bUWVhEwwdU0FWdKjnzIbXZetrrl1kpF3Ok3oOXMMKK9JDJ2ILTJeKXXFdUmXtBcvpRSXkk5WMqxbZo9nSON+rU3pYE0fXiEy7txXNcYnZKIhi0NqmW7CT5Fxk1a+v98xbkvr1Sk4VoisM6jDOSKXw8CMob655ESi0dCMby+SCX06mN0fgOfEIFx+svbm/PruZnUxVbj9e95lolhZIfibsVoAJ4Yk7Q2bSoDmtKc875py2FYGabQKVDBrYWOEhBN+qRaAu9gevtis4F/nsdVou1qA1BDcCZ/IWPCjwIPVl9n0C8qmQREBtfKd6aWmal1E43+S4WDQPMyREehCYo2XxoFz+M5N47GF9gj8IaE5pVGnYVtIvDgq6MlRDqk0Pc7toRqYJErKn4hTCQWD3CDFqI0Z49r6WABQ17+Wa8KFOrRSnHld1Y8WmzBcLJJPG2/S6Z2jsAhwL/eQGL0EKpNM61mShjzTau1FFNLgp1vhc5RiMgIrKCHity3XGlGUGQDu4+8ubvRswyEk6Hq8IxgUt2wdiHPzCtwlsxDTffiuaYkdlLH/h0mZAg5tWogS1JFULArROQdSVDCsQjFH0gjHlMw93/ktf1jBMec6Qjz5lNIpFlEzBeeSH6T8bzJOroTp7BClF92hpc/VQikexFU4AXVsztuNqdMfc6+2o7yUNFBViclJpjVdwnQIKWMhGKdTIl5SbOnw6fG/T1xYQcSswyKm5C7XL8/v1KSPEHWMcuWaPXNUFIAU5knLq/ic2vZpatu8ZGX/3vOKh1STF80yvRVFNz7ZtxvArCXZ4qQuH6essJS9K92AXHVzSmZiMmKDIwXZZOjlpN5R8L/hg10hIFbsa/vyaqXggx6yaqlYWmTZbjfSi4Th5L8+HebrBQw3am9zr3RhXXMYDAQ5bBYetR5W1b4CEs287iyd4DTZIt2eREzt/l2xc1kq9TTQtG98YmWfpaX5SpmHiZrUsG0I1XQeCbwHMi7f3sieRvLFBAK0R6dD3P2A1WDqMAZFWGPkIXyvW8mSC/SvPqGG0KwbcaNSkhm/TugWltY4AXP2WA1oCJyO0RoBf/h9VyprZ7QqqDNKN5k+s9Pjndf92cGjc72V2cL/q6IhF3QuPZE5dDdHIqz8Y1r8/OOYVST4E7Jk1K72Ch9DkVc5s1zL6il8APirYL9DtOllRb5rhZC8bGCXb2Zeagki8WMoeufuKLK5KuSxtrMwOf7mdR5jhh7iOiaBR23sB8dBL6/EUp+YplJQFiTbBoSKZc/2gQMhZxVfMeACf3wf8VqEfEiNwZYo1OqpULtY+RMS8Ff9M0QWrM8uUuNeUL9M1fi5N8JAC/Mp9i931rzMlVQQD9gNAAyPX0=
```

Comment:
Do we need these files (`bootstrap.txt` and `update_00290.txt`)? Why can't we read the same data from the eth2-light-client-updates repo?

Reply:
They are used only in `build_deploy_malaga.sh` script (which I committed because it is pretty helpful for debugging). It does not matter whether moving them in eth2-light-client-updates repo or even deleting them, because we have a `ts` function for converting our data from eth2-light-client-updates repo to this format - `jsonToSerializedBase64`

Reply:
Yes, I meant that you can convert and use the existing json files from the other repo.